tests/test_test_tools/test_test_contract.py

```diff
@@ -0,0 +1,225 @@
+            self.sign_send_confirm_receipt_multisig_transaction(
+                tx, self.ledger_api, [self.deployer_crypto],
+            )
+            assert str(e) == "Transaction digest not found!"
+        is_transaction_settled_mock.assert_not_called()
+        get_transaction_receipt_mock.assert_not_called()
+        send_signed_transaction_mock.assert_called_once_with(tx)
+        sign_transaction_mock.assert_called_once_with(tx)
+
+    @mock.patch.object(FetchAICrypto, "sign_transaction", return_value="tx")
+    @mock.patch.object(FetchAIApi, "send_signed_transaction", return_value="tx_digest")
+    @mock.patch.object(FetchAIApi, "get_transaction_receipt", return_value=None)
+    @mock.patch.object(FetchAIApi, "is_transaction_settled")
+    def test_sign_send_confirm_receipt_multisig_transaction_receipt_not_found(
+        self,
+        is_transaction_settled_mock,
+        get_transaction_receipt_mock,
+        send_signed_transaction_mock,
+        sign_transaction_mock,
+    ):
+        """Test the sign_send_confirm_receipt_multisig_transaction static method: receipt not found."""
+        tx = "tx"
+        sleep_time = 0
+        with pytest.raises(ValueError) as e:
+            self.sign_send_confirm_receipt_multisig_transaction(
+                tx, self.ledger_api, [self.deployer_crypto], sleep_time=sleep_time
+            )
+            assert str(e) == "Transaction receipt not found!"
+        is_transaction_settled_mock.assert_not_called()
+        get_transaction_receipt_mock.assert_called_with("tx_digest")
+        send_signed_transaction_mock.assert_called_once_with(tx)
+        sign_transaction_mock.assert_called_once_with(tx)
+
+    @mock.patch.object(FetchAICrypto, "sign_transaction", return_value="tx")
+    @mock.patch.object(FetchAIApi, "send_signed_transaction", return_value="tx_digest")
+    @mock.patch.object(
+        FetchAIApi, "get_transaction_receipt", return_value={"raw_log": "log"}
+    )
+    @mock.patch.object(FetchAIApi, "is_transaction_settled", return_value=False)
+    def test_sign_send_confirm_receipt_multisig_transaction_receipt_not_valid(
+        self,
+        is_transaction_settled_mock,
+        get_transaction_receipt_mock,
+        send_signed_transaction_mock,
+        sign_transaction_mock,
+    ):
+        """Test the sign_send_confirm_receipt_multisig_transaction static method: receipt not valid."""
+        tx = "tx"
+        sleep_time = 0
+        with pytest.raises(ValueError) as e:
+            self.sign_send_confirm_receipt_multisig_transaction(
+                tx, self.ledger_api, [self.deployer_crypto], sleep_time=sleep_time
+            )
+            assert str(e) == "Transaction receipt not valid!\nlog"
+        is_transaction_settled_mock.assert_called_with({"raw_log": "log"})
+        get_transaction_receipt_mock.assert_called_with("tx_digest")
+        send_signed_transaction_mock.assert_called_once_with(tx)
+        sign_transaction_mock.assert_called_once_with(tx)
```

Comment:
Please add a simple test for `sign_send_confirm_receipt_transaction` asserting it calls `sign_send_confirm_receipt_multisig_transaction`.